lib/timeline/Core.js

```diff
@@ -161,6 +161,35 @@ Core.prototype._create = function (container) {
   this.dom.root.addEventListener('mousewheel', onMouseWheel);
```

Comment:
I would suggest to use the chance an change the existing mousewheel handler to something more generic:
```js
if (this.dom.root.addEventListener) {
	// IE9, Chrome, Safari, Opera
	this.dom.root.addEventListener("mousewheel", MouseWheelHandler, false);
	// Firefox
	this.dom.root.addEventListener("DOMMouseScroll", MouseWheelHandler, false);
} else {
	// IE 6/7/8
	this.dom.root.attachEvent("onmousewheel", MouseWheelHandler);
}
```